.gitignore

```diff
@@ -1 +1,5 @@
+*.swp
+
```

Comment:
We likely need to add other things, like `dataset`, etc.

Reply:
Agree, I think, the best would be to fill up the `.gitignore` as we work on the project.